modules/git/commit_info.go

```diff
@@ -7,5 +7,5 @@ package git
 type CommitInfo struct {
 	Entry         *TreeEntry
 	Commit        *Commit
```

Comment:
Looks like `*Commit` could be removed.

Reply:
The `Commit` was removed from `CommitSubModuleFile` because it was never used.